### PR TITLE
 Replace release button if up to date 

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Pass production information to the template and conditionally display the release button based on the commit ids. Added format-python npm script to aid linting.

## QA
- Run the site
- Go to http://0.0.0.0:8103/
- Check that some sites have "Up to date" where the release button used to be

FIxes https://github.com/canonical-web-and-design/releases.demo.haus/issues/15

### CODE REVIEW

**Comment by nottrobin** [comment]
Seeing a lot of:

image

Why might that be?

**Comment by anthonydillon** [comment]
@nottrobin fixed and ready for your eyes again

File: templates/index.html [hunk 1]
```diff
@@ -290,34 +291,38 @@ <h1>Web &amp; design team deployment portal</h1>
...
+          commit.innerHTML = `<a href="https://github.com/${domainRepositories[domainJson['staging']['domain']]}/commit/${stagingCommitID}">${stagingCommitID}</a>`;
+          humanTime.textContent = domainJson['staging']['humanTime'];
+        humanTime.dataset.sort = domainJson['staging']['epochTime'];
+          if (release) {
+            if (stagingCommitID !== productionCommitID) {
+              release.innerHTML = `<a style="width: 100%" class="p-button--neutral" href="https://jenkins.canonical.com/webteam/securityRealm/commenceLogin?from=%2Fwebteam%2Fjob%2Fdeploy-to-${domainJson['staging']['domain']}%2Fparambuild%2F%3Fimage_tag%3D${domainJson['staging']['imageTag']}">` +
+                                  `  Release 🚀` +
+                                  `</a>`;
+            } else {
+              release.innerHTML = '<a style="width: 100%" class="p-button--neutral" href="#" disable="disable">Up to date 👍</a>';
```
Comment (nottrobin) [inline templates/index.html]: Couldn't you make this a `<button disabled="true">`? 'Cos a link to `#` is a somewhat misleading experience.

Comment (anthonydillon) [reply templates/index.html]: Good spot, forgot I wasn't finished there. Done

**Pull request merged**